src/api/crosscutRequests.js

```diff
@@ -281,15 +281,89 @@ export const fetchSupportedBoundaries = async () => {
         authorization: getToken(),
       },
     }).json();
+    // sort/organize the boundary list
+    const boundaryList = Object.values(resp.boundaryList);
 
-    return resp.boundaryList
-      .filter((bound) => bound.featureFlags.includes('all'))
-      .filter((bound) => bound.entireCountry === true)
-      .sort((a, b) => {
-        if (a.countryName > b.countryName) return 1;
-        if (a.countryName < b.countryName) return -1;
+    const entireCountries = boundaryList.filter(
```

Comment:
Add a filter for entire country and partial country to only include feature flags that include `all`

Reply:
[f61a280](https://github.com/crosscutio/dhis2-crosscut-app/pull/59/commits/f61a2803ebba6c7b375140a3b98b216405c794a3)